07-knitr-R.Rmd

```diff
@@ -11,31 +11,44 @@ source("tools/chunk-options.R")
 
 > ## Learning Objectives {.objectives}
 >
-> * To discover `knitr` and the generation of dynamic reports
+> * Understand the value of `knitr`: Generate dynamic documents that include text, code, and results.
+> * Control basic formatting using markdown syntax.
+> * Be able to create, edit, and compile an .Rmd document including code chunks and inline code.
 
-`Knitr` is an R package that makes your code neat, pretty, and shows your notes, code, and output simultaneously in an `html` document. You create these documents in `.Rmd` files. You can write in `LateX` or `md`. 
+`knitr` is an R package that allows you to organize your notes, code, and results in a single document. It's a great tool for "literate programming" -- the idea that your code should be readable by humans as well as computers! It also keeps your writing and results together, so if you collect some new data or change how you clean the data, you just have to re-compile the document and you're up to date!
 
-knitr extracts R code in the input document (.Rmd), evaluates it and writes the results to the output document (html). There are two types of R code: chunks (code as separate paragraphs) and inline R code.
+You write `knitr` documents in a simple plain text-like format called markdown, which allows you to format text using intuitive notation, so that you can focus on the content you're writing. But you still get a well-formatted document out. In fact, you can turn your plain text (and R code and results) into an html file or, if you have an installation of LaTeX and Pandoc on your machine, a pdf, or even a Word document (if you must!).
+
+To get started, install the `knitr` package.
 
 ```{r, eval=FALSE}
 install.packages("knitr")
 ```	
-```{r}
-library(knitr)
-```    
 
-Restart RStudio. 
-Open a new `Rmd` file. 
+When you click on File -> New File, there is an option for "R Markdown...". Choose this and accept the default options in the dialog box that follows (but note that you can also create presentations this way). Save the file and click on the "Knit HTML" button at the top of the script. Compare the output to the source.
 
-In Rmd, anything between lines that start and end with triple quotes ``` will be run as R code.
+> ## Challenge - Explore formatting text in markdown {.challenge}
+>
+> Visit <http://rmarkdown.rstudio.com/authoring_basics.html> and briefly check out some of the formatting options. 
+>
+> In the example document add
+> 
+> * Headers using `#`
+> * Emphasis using astericks:  \*italics\* and \*\*bold\*\*
+> * Lists using `*` and numbered lists using `1.`, `2.`, etc.
+> * **Bonus:** Create a table
 
-```{r}
-summary(cars)
-```
+Markdown also supports LaTeX equation editing. We can display pretty equations by enclosing them in `$`. E.g. `$\alpha = \dfrac{1}{(1 - \beta)^2}$` renders as: $\alpha = \dfrac{1}{(1 - \beta)^2}$.
```

Comment:
The equation doesn't render properly using our pandoc-build process. I created an [Issue](https://github.com/swcarpentry/lesson-template/issues/285) for the lesson-template repo to investigate.